description.md

```diff
@@ -34,6 +34,50 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Practical Sessions
 
+Practical sessions are represented by hands-on activities that learners perform in order to acquire practical experience.
+These activities are typically supervized by a teacher that proposes the practical exercices and evaluates the learner's performance.
+The purpose of the evaluation is mainly to provide constructive feedback, as opposed to grading the student.
```

Comment:
```suggestion
The purpose of the evaluation is mainly to provide constructive feedback and support, as opposed to grading the student.
```

Reply:
I would refrain from using the word "constructive". It doesn't actually say anything.

Also, I wouldn't say the primary purpose is feedback. The primary purpose is to grain practical skills, to learn by doing.

Reply:
The purpose of the evaluation is to provide feedback. The purpose of the practical session is to learn by doing as mentioned in the leading sentence.

Reply:
@gabrielmocanu also, the purpose of the evaluation is not to provide support, but to provide feedback.